app/directives/search/search-result.html

```diff
@@ -78,7 +78,7 @@
 			aichiTarget_ss -->
 
 			<span ng-repeat="hostG in document.hostGovernments_EN_txt" > <span class="label label-default" > {{hostG}}</span>&nbsp; </span>
-			<span ng-repeat="hostG in document.thematicArea_EN_txt"><span class="label label-info small" >{{hostG}}</span>&nbsp; </span>
+			<span ng-repeat="hostG in document.thematicArea_EN_txt | limitTo:5"><span class="label label-info small" >{{hostG}}</span>&nbsp; </span>
```

Comment:
can you add more button if there are more than 5 items and show the remaining when clicked. it will otherwise confuse users  why there are only 5 items

Reply:
I added an ellipsis instead. In my mind, if we get into hide/showing on a search result I think there is no limit.  Everything could/should be shown and hidden to be consistent not just the themes. The search result is meant to be a preview and to see more they click the full doc.

I think the ellipsis solves any confusion one may have and is consistent with the way the description/summary is truncated.  Do you agree?